plugins/modules/clustering/consul/consul.py

```diff
@@ -120,6 +120,13 @@
             Similar to the interval this is a number with a s or m suffix to
             signify the units of seconds or minutes e.g C(15s) or C(1m). If no suffix
             is supplied, C(m) will be used by default e.g. C(1) will be C(1m)
+    tcp:
+        type: str
+        description:
+          - checks can be registered with a TCP port. This means that consul
+            will check if the connection attempt to that port is successful (ie - the port is currently accepting connections).
+            I(Interval) must also be provided with this option.
```

Comment:
```suggestion
            I(interval) must also be provided with this option.
```

Reply:
Maybe also mention that the value for this option must be of the form `<host>:<port>`?